devops/openshift/_legal-api.dc.json

```diff
@@ -109,23 +455,22 @@
                         "type": "ConfigChange"
                     }
                 ],
-                "replicas": "${REPLICAS}",
+                "replicas": "${{REPLICA_MIN}}",
```

Comment:
Double brackets!

Reply:
Double braces allow for input of non-string values, the quotes will get replaced with the integer value. 

https://docs.openshift.com/container-platform/3.11/dev_guide/templates.html#writing-parameters